src/main/java/edu/cornell/kfs/krad/service/impl/CuKualiExceptionIncidentServiceImpl.java

```diff
@@ -0,0 +1,28 @@
+package edu.cornell.kfs.krad.service.impl;
+
+import org.kuali.kfs.krad.service.impl.KualiExceptionIncidentServiceImpl;
+
+public class CuKualiExceptionIncidentServiceImpl extends KualiExceptionIncidentServiceImpl {
+    
+    //Cornell Customization: Base code implemenation override required due to AppSMTP 
+    //                       restriction where From email address must be a registered EGA.
+    @Override
+    protected String getFromAddress() {
+    /* Base code at time of cutomization creation
```

Comment:
Some of these comments should be cleaned up. The original code in commented form can be removed, since we can go to the superclass to view it if needed. It's also not clear about how "kr.incident.mailing.list" relates to this unless you look at the bean file.

In addition, you need to override the appropriate service bean so that it will use this new subclass. (I think "cf.kradExceptionIncidentService" is the bean name.)

Reply:
Comments clean up. Thanks for catching the bean definition.